about/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-layout: page
+layout: Page

```

Comment:
There's no need to capitalise this word, you can leave it as it originally was. Once that change is reverted, I can approve the pull request,

Reply:
okay done